spec-examples/src/ntru_prime/ntru_prime.rs

```diff
@@ -0,0 +1,147 @@
+use hacspec::prelude::*;
+
+/// Struct to decide NtruVersion
+pub struct NtruVersion {
```

Comment:
We don't actually allow `struct` right now. I have to look into whether we want to change that. Let's keep it this way for now. That's easy to change later if necessary.